devcontainer/devcontainer.json

```diff
@@ -0,0 +1,64 @@
+// Reference: https://containers.dev/implementors/json_reference/
+{
+  "name": "Swift Package devContainer",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {
+      // Update 'SWIFT_VERSION' to change the Swift version in the container.
+      "SWIFT_VERSION": "6.2"
```

Comment:
With the latest change I think now this needs to be `latest` or commented out, otherwise this will need to be updated every 6 months on each repo